### DISCUSSION TITLE
Not plotting anymore. How to stop Chia to check for new plots?

### DISCUSSION THREAD

**Discussion by jmz-bit** [Chia-Network/chia-blockchain#9022] [Support]
I found this section in Chia config file.

plots_refresh_parameter:
    batch_size: 300
    batch_sleep_milliseconds: 1
    **interval_seconds: 3600**
    retry_invalid_seconds: 1200

I changed interval_seconds: to 3600 seconds from it default 120. 

What is the biggest value of this parameter? Can I set it to 86400 seconds (24h)?
I'm not sure about that high value as my guess is if this value will be to big could be ignored by app and then default option of 120 second could be used.

The point is to limit discs operations. I'm not creating new plots so it make no sense for the app to check for new plots every 2 minutes.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
